Formula/gopass-jsonapi.rb

```diff
@@ -0,0 +1,45 @@
+class GopassJsonapi < Formula
+  desc "Gopass Browser Bindings"
+  homepage "https://github.com/gopasspw/gopass-jsonapi"
+  url "https://github.com/gopasspw/gopass-jsonapi/archive/v1.11.1.tar.gz"
+  sha256 "3c1666cdbf78a73736b089c3188c06724de53a96e43cb1d82fedd7e9543c120f"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "gopass"
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-s -w -X main.version=#{version}"
+  end
+
+  test do
+    (testpath/"batch.gpg").write <<~EOS
+      Key-Type: RSA
+      Key-Length: 2048
+      Subkey-Type: RSA
+      Subkey-Length: 2048
+      Name-Real: Testing
+      Name-Email: testing@foo.bar
+      Expire-Date: 1d
+      %no-protection
+      %commit
+    EOS
+
+    begin
+      system Formula["gnupg"].opt_bin/"gpg", "--batch", "--gen-key", "batch.gpg"
+
+      system Formula["gopass"].opt_bin/"gopass", "init", "--path", testpath, "noop", "testing@foo.bar"
+      system Formula["gopass"].opt_bin/"gopass", "generate", "Email/other@foo.bar", "15"
+    ensure
+      system Formula["gnupg"].opt_bin/"gpgconf", "--kill", "gpg-agent"
+      system Formula["gnupg"].opt_bin/"gpgconf", "--homedir", "keyrings/live",
+                                                 "--kill", "gpg-agent"
+    end
+
+    assert_match version.to_s, shell_output("#{bin}/gopass-jsonapi --version")
+
+    msg='{"type": "query", "query": "foo.bar"}'
+    out = pipe_output("#{bin}/gopass-jsonapi listen", [msg.length].pack("L<")+msg)
+    assert_match %r{Email/other@foo.bar}, out
```

Comment:
```suggestion
    assert_match "Email/other@foo.bar", out
```
You don't use any regex tokens here, so I think you can just pass this as a string. Squash your commits together if you make this change, please. Thanks!

Reply:
Thanks. I have removed the regex token and squashed my commits.